src/main/java/network/brightspots/rcv/JsonParser.java

```diff
@@ -37,31 +38,32 @@ class JsonParser {
   // param: logsEnabled display log messages
   // returns: instance of the object parsed from json or null if there was a problem
   private static <T> T readFromFile(String jsonFilePath, Class<T> valueType, boolean logsEnabled) {
-    T createdObject;
-    try {
-      FileReader fileReader = new FileReader(jsonFilePath);
+    try (FileReader fileReader = new FileReader(jsonFilePath, StandardCharsets.UTF_8)) {
       ObjectMapper objectMapper = new ObjectMapper();
-      createdObject = objectMapper.readValue(fileReader, valueType);
+      return objectMapper.readValue(fileReader, valueType);
```

Comment:
as discussed in the issue, we're not allowed to have multiple returns :(

Reply:
Reverted.